### DISCUSSION TITLE
Apply past tags to repeated timers

### DISCUSSION THREAD

**Discussion by AetheresMMI** [mcndt/obsidian-toggl-integration#12] [Feedback]
Just a small bug I noticed using/testing this plugin over the past few days.

When retriggering past timers in the Obsidian interface, the name and project are properly applied, but the associated tags are lost and need to be applied manually.

**Comment by mcndt** [comment]
Thanks for your report! I'll try to fix this before the end of the week, should be pretty easy.

**Comment by mcndt** [comment]
Fixed in release 0.2.3, archiving conversation.
